simulator/src/plotArea.js

```
@@ -284,7 +346,7 @@ const plotArea = {
 
         for (var i = 0; i < globalScope.Flag.length; i++) {
             var plotValues = globalScope.Flag[i].plotValues;
-            var startHeight = getFlagStartY(i) + waveFormPadding;
+            var startHeight = getFlagStartY(i) + getWaveFormPadding();
```

Biome

[error] Shouldn't redeclare 'startHeight'. Consider to delete it or rename it.

'startHeight' is defined here:


(lint/suspicious/noRedeclare)